tests/hardhat/Prime/Prime.ts

```diff
@@ -190,7 +190,7 @@ async function deployProtocol(): Promise<SetupProtocolFixture> {
     ],
     {
       constructorArgs: [false, 10512000],
-      unsafeAllow: "constructor",
+      unsafeAllow: ["constructor", "internal-function-storage"],
```

Comment:
Why are we adding this? Could we add a comment where we are adding this flag, or at least a commit message with a body explaining the reason?

Reply:
[51c0702](https://github.com/VenusProtocol/venus-protocol/pull/506/commits/51c070278168a0a65789c3b9e6320a1f57d5876c)